harness/attributes/common.yml

```diff
@@ -31,10 +31,13 @@ attributes.default:
     # If using gitops helm chart repositories, it's recommended to put this configuration
     # in the values.yml in there now instead of project application repositories
     image_pull_config: = @('docker.config')
-    registry_url: = get_docker_registry(@('docker.repository'))
```

Comment:
should we keep this attribute for backwards compatibility? not sure if any Go apps are using this attribute in overrides, or whether we even consider that a BC break

Reply:
Since there was no tag and presumably only my OLP work uses it, I'd like to just upgrade OLP. It's not a huge pain to have the BC though, but gets untidy